fdbserver/storageserver.actor.cpp

```diff
@@ -4384,12 +4413,15 @@ ACTOR Future<Void> metricsCore(StorageServer* self, StorageServerInterface ssi)
 			                                             self->thisServerID.second() ^ self->ssPairID.get().second()));
 		                               }
 
-                                       // These are the size of mutations since last "StorageMetrics" emission. It's
-		                               // easy to get bandwidths by dividing by "Elapsed".
-		                               te.detail("SSFetchKeysBytes", self->fetchKeysBytes);
-		                               te.detail("SSUpdateMutationBytes", self->updateMutationBytes);
-		                               self->fetchKeysBytes = 0;
-		                               self->updateMutationBytes = 0
+		                               // The size of data processed for data movement and actual writes. They are
+		                               // roughly comparable, but the overhead is counted differently.
+		                               logCounterAndResetInterval(te, "BytesFetched", self->counters.bytesFetched);
```

Comment:
You don't need to log these here, by virtue of being counters declared on the `CounterCollection` they will be logged as part of `StorageMetrics`.

Reply:
Great to know that! I also see `CounterCollection::logToTraceEvent` resets the interval.